src/app/selectors/customAttributes.js

```diff
@@ -9,6 +10,7 @@ const placeKeys = Object.keys(place)
 export const allCustomAttributesSelector = state => state.customAttributes
 export const characterCustomAttributesSelector = state => state.customAttributes.characters
 export const placeCustomAttributesSelector = state => state.customAttributes.places
+export const scenesCustomAttributesSelector = state => state.customAttributes.scenes
```

Comment:
all the changes to this file will need to be mirrored to the pltr repo (and that change above from the selectors/cards.js file)

Reply:
Ahh.  You mentioned that some of the logic hadn't been extracted, but it didn't quite click for me that there was duplication.
Let me see about migrating anything I've touched.